### ISSUE TITLE
Add manual examples

### ISSUE THREAD

**Issue by olexandr-konovalov** [gap-packages/RepnDecomp#5]
The manual looks good in general, but it does not have any examples. Having examples improves its readability and usability - the reader may see immediately sample inputs and outputs, and can cope and paste the input into their code. Please consider adding examples - at least a "Sample session with RepnDecomp" for the start; ideally, to most of the mansections.

**Comment by kaashif** [comment]
Yes, this is very important. You can even see there is an `examples/` directory where I said I would add some examples but I never got around to it. I will delete that directory and instead add some examples to the manual. Thanks for reminding me about this.

**Comment by olexandr-konovalov** [comment]
Great! You can also extract and test manual examples, so you're this will not only improve the documentation, but also may improve regression testing and code coverage.
